src/scripts/main.js

```diff
@@ -22,6 +22,23 @@ const tree = document.querySelector('#tree');
 
 function createTree(element, data) {
   // WRITE YOUR CODE HERE
+  const ul = document.createElement('ul');
+
+  element.appendChild(ul);
+
+  for (const entry of Object.entries(data)) {
+    const li = document.createElement('li');
+
+    li.textContent = entry[0];
+    ul.appendChild(li);
+
+    if (Object.keys(entry[1]).length > 0) {
+      const newUl = document.createElement('ul');
+
+      element.appendChild(newUl);
```

Comment:
There is a mistake here. The `newUl` should be appended to the `li` element, not the `element`. This ensures that the nested list is correctly placed under the corresponding list item.